packages/web-components/src/components/card/__stories__/card.stories.ts

```diff
@@ -98,7 +98,11 @@ export default {
   decorators: [
     story => html`
       <div class="dds-ce-demo-devenv--simple-grid dds-ce-demo-devenv--simple-grid--card">
-        ${story()}
+        <div class="bx--row dds-ce-demo-devenv--grid-row">
+          <div class="bx--col-sm-4 bx--col-md-3 bx--col-lg-12 bx--col-xlg-8 bx--no-gutter">
+            ${story()}
+          </div>
+        </div>
```

Comment:
Hi @RaphaelAmadeu thank you for the adjustment; Would you be able to try doing this adjustment using `grid-column`, like: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/072e68f/packages/web-components/.storybook/container.scss#L77-L101